### ISSUE TITLE
Fire Mage Unable to Learn any Glyphs

### ISSUE THREAD

**Issue by zeus2422** [dalaranwow/dalaran-wow#5727]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
My fire mage is unable to learn the major glyph "Glyph of Living Bomb" (id 45737) or "Glyph of Molten Armor" (id 42751) or "Glyph of Slow Fall" (id 43364) when right clicking on the glyphs in my bags. Seems like the entire glyph interface on "N" does not work. for me

**Expected behaviour**: Tell us what should happen instead
Should be able to successfully learn the glyphs and equip them.

**Steps to reproduce the problem**:

1. Launch WoW
2. Log in to the game
3. Right click on the glyphs in bags

**Include proofs for this behaviour**
https://gyazo.com/54d5c72f6eac5f0845005a11114ee6a4

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
45737
42751
43364
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Most likely an interface, addon or cache issue. Either restart your client (if you didn't already) or clear your wtf folder for the character in question.

**Comment by Golganeth** [comment]
Sometimes addons or corrupt UI elements can prevent interaction with your UI like this. Are you having the issue with just one character or with all of them?

Disable all of your addons check if it works. If that was indeed the problem I'd suggest you to turn them n one by one, so you can easily detect which addon was giving you trouble.

If the issue persists:
- As it was suggested you can clear your Cache folder (this might fix small minor issues and it's pretty quick to do). Try also clearing your WTF folder, you will lose your settings but it can potentially fix the problem as a last option.
- At this point if nothing has worked, it's most likely that your client has corrupted files so you should re-install the game from scratch. You can get our game client [here](https://www.dalaran-wow.com/game-client/).

Closing up since the issue is on your side.
